oshc/main/templates/registration/registration_form.html

```diff
@@ -23,23 +23,22 @@ <h1 class="form-title text-center">{% trans 'Sign Up Here' %}</h1>
                                     {% trans 'Password:' %}</label>
                 <input class="form-control" id="id_password1" name="password1"  type="password" onkeyup="checkPasswordStrength(this.value)" required/><br>
                 
-       <meter id="abc" min="0" low="30" optimum="100" high="60" max="100" value=""></meter>
+       <meter id="meter" min="0" low="30" optimum="100" high="60" max="100" value=""></meter>
```

Comment:
@vaibhavsingh97  Can't connect script.js to html, any guidence ? how to connect Script which are not in same file. Thank you